### DISCUSSION TITLE
Logout from all devices using next-auth v5?

### DISCUSSION THREAD

**Discussion by igmtink** [nextauthjs/next-auth#9679] [Help]
I'm using next-auth v5 on my next.js 14 project, also I'm using "JWT" strategy and prisma adapter, can anyone give me an idea or example logic to implement the "logout from all devices"?

**Comment by pedro757** [comment]
I'd delete every session for the user in the sessions table.

**Comment by taylor-lindores-reeves** [comment]
+1
